internal/cmd/mockery.go

```diff
@@ -322,5 +356,16 @@ func (r *RootApp) Run() error {
 		}
 	}
 
+	// The loop above could exit early, so sometimes warnings won't be shown
+	// until other errors are fixed
+	for packageName := range missingMap {
+		for iface := range missingMap[packageName] {
+			log.Warn().
+				Str(logging.LogKeyInterface, iface).
+				Str(logging.LogKeyPackagePath, packageName).
+				Msg("no such interface")
```

Comment:
```suggestion
				Msg("interface not found in source")
```

Just to be a bit more explicit.

Reply:
sure